src/index.js

```diff
@@ -159,11 +160,12 @@ export default declare(({
         }
       },
       ExportNamedDeclaration(path, state) {
-        const { node } = path;
+        const { node, scope } = path;
         if (node.specifiers.length > 0 && node.specifiers[0].local.name === 'default') {
           const exportName = node.specifiers[0].exported.name;
           const filename = parseFilename(node.source.value).name;
-          applyPlugin(exportName, node.source.value, path, state, true, filename);
+          const newPath = applyPlugin(exportName, node.source.value, path, state, true, filename);
+          if (newPath) scope.registerDeclaration(newPath);
```

Comment:
```suggestion
          if (newPath) {
            scope.registerDeclaration(newPath);
          }
```
never omit curly braces